index.js

```diff
@@ -1,55 +1,199 @@
 const state = {
   items: [
     {
-      id: "001-beetroot",
-      name: "beetroot",
+      id: '001-beetroot',
+      name: 'beetroot',
       price: 0.35
     },
     {
-      id: "002-carrot",
-      name: "carrot",
+      id: '002-carrot',
+      name: 'carrot',
       price: 0.35
     },
     {
-      id: "003-apple",
-      name: "apple",
+      id: '003-apple',
+      name: 'apple',
       price: 0.35
     },
     {
-      id: "004-apricot",
-      name: "apricot",
+      id: '004-apricot',
+      name: 'apricot',
       price: 0.35
     },
     {
-      id: "005-avocado",
-      name: "avocado",
+      id: '005-avocado',
+      name: 'avocado',
       price: 0.35
     },
     {
-      id: "006-bananas",
-      name: "bananas",
+      id: '006-bananas',
+      name: 'bananas',
       price: 0.35
     },
     {
-      id: "007-bell-pepper",
-      name: "bell pepper",
+      id: '007-bell-pepper',
+      name: 'bell pepper',
       price: 0.35
     },
     {
-      id: "008-berry",
-      name: "berry",
+      id: '008-berry',
+      name: 'berry',
       price: 0.35
     },
     {
-      id: "009-blueberry",
-      name: "blueberry",
+      id: '009-blueberry',
+      name: 'blueberry',
       price: 0.35
     },
     {
-      id: "010-eggplant",
-      name: "eggplant",
+      id: '010-eggplant',
```

Comment:
why this quota sign have been changed

Reply:
i got this he used everything in single quota

Reply:
I think we should blame prettier